content/javascript/concepts/sets/terms/add/add.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: '.add()'
+Description: 'Inserts a value into a Set (if it is unique) and returns the updated Set object.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Sets'
+  - 'Objects'
+  - 'Collections'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`.add()`** method inserts a value into a `Set` (if it is unique) and returns the updated `Set` object.
+
+## Syntax
+
+```pseudo
+set.add(value)
+```
+
+The `value` in `set` is unique and of a valid [data type](https://www.codecademy.com/resources/docs/javascript/data-types).
+
+## Codebyte Example
+
+```codebyte/javascript
+const mySet = new Set();
+
+mySet.add(1);
+mySet.add("text");
+console.log(mySet);
+
+mySet.add(1);
+console.log(mySet)
```

Comment:
I know the compiler adds the semicolon. But for documentation purposes, I think they should always be added.
```suggestion
console.log(mySet);
```